blocks/search/search.css

```diff
@@ -14,14 +14,14 @@
   padding: 0.5em;
   border-radius: 4px;
   border: 1px solid var(--dark-color);
```

Comment:
Looks like this line is redundant, since line 23 styles over it.

Reply:
You sure? the selectors are different

Reply:
missed this but it is updated now